ui/src/app/pages/access/maybe-module.tsx

```diff
@@ -47,7 +46,6 @@ export const MaybeModule = ({
     [AccessModule.TWO_FACTOR_AUTH, () => () => setShowTwoFactorAuthModal(true)],
     [AccessModule.RAS_LINK_LOGIN_GOV, () => redirectToRas],
     [AccessModule.IDENTITY, () => redirectToRas],
-    [AccessModule.ERA_COMMONS, () => redirectToNiH],
```

Comment:
@jmthibault79 I removed this case here...but want to check with you that this is ok

Reply:
yes this should be fine to remove